edit_book.php

```diff
@@ -0,0 +1,44 @@
+<?php
+include 'database.php';
+
+if (isset($_GET['id'])) {
```

Comment:
**Observations:** The script handles both fetching and updating book data using procedural code, which mixes database logic with HTML rendering.

**Recommendations:**
- **Separate Concerns:**
  - Use an MVC structure to separate logic. Implement a `Book` model and a `BookRepository` for database operations.
  - Move HTML to a separate view file.

- **Use Prepared Statements for Updates:**
  - Use prepared statements to update book information safely.
  - Example:
    ```php
    $stmt = $connection->prepare("UPDATE books SET title = ?, author = ?, year = ? WHERE id = ?");
    $stmt->bind_param("ssii", $title, $author, $year, $id);
    $stmt->execute();
    ```

- **Validate Input Data:**
  - Ensure that input data is validated before processing to prevent invalid entries.